### PR TITLE
Allow optparse-applicative-0.17

### PR DESCRIPTION
**Pull request by mrkkrp**

### CODE REVIEW

**Comment by mrkkrp** [comment]
Hi @sestrella, would it be possible for me to re-gain write access to this repo? I see that the master branch has become protected, which means that every change has to receive an approval now. It does feel a bit like overkill given that I have been maintaining this project for the last 6 years virtually alone. If I have to obtain external approvals in order to continue maintaining this project, it is going to increase the friction of working on it for me. Which is fine, if Stack Builders can guarantee quick feedback. I just want to figure out for now if imposing approval on my work has been a conscious decision or a side-effect of apply the same settings to all Stack Builders repos.

**Review by sestrella — APPROVED** [review]
LGTM - About the protected branch, let me follow up about it

**Pull request merged**
